tests/aws/services/kms/test_kms.py

```diff
@@ -6,9 +6,15 @@
 from datetime import datetime
 from random import getrandbits
 
+import cbor2
 import pytest
+from asn1crypto import cms
 from botocore.config import Config
 from botocore.exceptions import ClientError
+from Crypto.Cipher import AES, PKCS1_OAEP
+from Crypto.Hash import SHA256
+from Crypto.PublicKey import RSA as CryptoRSA
+from Crypto.Util.Padding import unpad as crypto_unpad
```

Comment:
Same as suggested in the `crypto.py` most of this can be done with `cryptography` library.